sdks/actors/client/src/client.ts

```diff
@@ -126,16 +121,14 @@ export class Client {
 		}) as ActorHandle;
 	}
 
-	async #createHandle(
-		tags: ActorTags,
-		opts?: WithTagsOpts,
-	): Promise<ActorHandleRaw> {
+	async #createHandle(tags: ActorTags, opts?: WithTagsOpts): Promise<ActorHandleRaw> {
 		const create = opts?.create ?? {
 			tags,
 			buildTags: {
 				name: tags.name,
 				current: "true",
 			},
+			region: (await this.#resolveRegion()).id,
```

Comment:
This should be cached locally for 48h. Otherwise, this will add significant latency to creating new actors.